frontend/src/components/dialogs/MemberDialog.vue

```diff
@@ -205,17 +239,26 @@ export default {
       return this.type === 'updateuser' || this.type === 'updateservice'
     },
     roleItems () {
-      return MEMBER_ROLE_DESCRIPTORS
+      return filter(MEMBER_ROLE_DESCRIPTORS, role => {
+        return !role.notEditable
+      })
     },
     nameLabel () {
       return this.isUserDialog ? 'User' : 'Service Account'
     },
     nameHint () {
       if (this.isUserDialog) {
-        return 'Enter the username that should become a user of this project'
+        return 'Enter the username that should become a member of this project'
       }
       if (this.isServiceDialog) {
-        return 'Enter the name of a Kubernetes Service Account'
+        return 'Enter a valid Kubernetes Service Account name'
```

Comment:
this sounds like a validation error text

```suggestion
        return 'Enter service account name'
```